dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jaxwsweb-2.3.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jaxwsweb-2.3
+visibility=private
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.jaxws-2.3))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.servlet-3.0)(osgi.identity=com.ibm.websphere.appserver.servlet-3.1)(osgi.identity=com.ibm.websphere.appserver.servlet-4.0)))"
```

Comment:
isn't jaxws-2.3 associated with Java EE 8? Should it be compatible with EE 6/7 servlet features?

Reply:
@neuwerk